staging/src/k8s.io/apimachinery/pkg/util/net/interface.go

```diff
@@ -268,22 +331,33 @@ func (_ networkInterface) Interfaces() ([]net.Interface, error) {
 	return net.Interfaces()
 }
 
-func chooseHostInterfaceFromRoute(inFile io.Reader, nw networkInterfacer) (net.IP, error) {
-	routes, err := getRoutes(inFile)
+// getAllDefaultRoutes obtains IPv4 and IPv6 default routes on the node. If unable
+// to get IPv4 default routes, we return an error. If unable to get IPv6 routes,
+// we'll just use the IPv4 route information. If no routes are found, an error is
+// returned.
+func getAllDefaultRoutes() ([]Route, error) {
+	routes, err := v4File.extract()
 	if err != nil {
 		return nil, err
```

Comment:
See response above. I think it is OK, but let me know if there is some different wordsmithing needed.